nixos/modules/virtualisation/podman-dnsname.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (lib)
+    mkOption
+    mkIf
+    types
+    ;
+
+  cfg = config.virtualisation.podman.dnsname;
+
+in
+{
+  options = {
+    virtualisation.podman.dnsname = {
```

Comment:
I'm not necessarily :-1:  on this but it doesn't seem quite right? By itself It's an option to add the `dnsname` plugin path to `containersConf` which we'll also have (manually) via `containersConf.cniPlugins`?

Could we do `virtualisation.podman.defaultNetwork.dnsname.enable` which adds the config and sets the plugin path?

I think that may also better align the options with your comment about a possible future rename?

Reply:
Done.